src/Utils.ts

```diff
@@ -0,0 +1,11 @@
+export enum Environment {
+    Development,
+    Staging,
+    Production,
+}
+
+export function environmentToContractNameInterfix(env: Environment): string {
```

Comment:
Nit: the word interfix denotes a word connecting two words, which itself does not have any meaning.
IMO these would better be described as infixes.

Reply:
Thank you, this is fixed